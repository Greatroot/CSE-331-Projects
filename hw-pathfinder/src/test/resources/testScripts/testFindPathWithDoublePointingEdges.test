# Tests FindPath.findSmallestPath() with edges that point both ways.

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n4 n3 1.0
AddEdge g n3 n2 2.0
AddEdge g n2 n1 3.0
AddEdge g n1 n2 6.0
AddEdge g n2 n3 10.0
AddEdge g n3 n4 4.0

FindPath g n1 n4