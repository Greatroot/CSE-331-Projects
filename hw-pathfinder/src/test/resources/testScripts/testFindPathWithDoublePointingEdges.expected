# Tests FindPath.findSmallestPath() with edges that point both ways.

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge 1.000 from n4 to n3 in g
added edge 2.000 from n3 to n2 in g
added edge 3.000 from n2 to n1 in g
added edge 6.000 from n1 to n2 in g
added edge 10.000 from n2 to n3 in g
added edge 4.000 from n3 to n4 in g

path from n1 to n4:
n1 to n2 with weight 6.000
n2 to n3 with weight 10.000
n3 to n4 with weight 4.000
total cost: 20.000